vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/endpoints/openapi/openapi.go

```diff
@@ -133,7 +133,8 @@ func gvkConvert(gvk schema.GroupVersionKind) v1.GroupVersionKind {
 	}
 }
 
-func friendlyName(name string) string {
+// FriendlyName returns REST resource friendly naming
```

Comment:
Why this change? I don't see this being used (I might missed it), upstream does not have it.